File: src/main/java/com/shiminfxcvii/turing/model/dto/UploadFileDTO.java
package com.shiminfxcvii.turing.model.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Getter;
import lombok.Setter;

/**
 * UploadFileDTO
 *
 * @author ShiminFXCVII
 * @since 2023/4/3 10:52
 */
@Getter
@Setter
public class UploadFileDTO {

    /**
     * 文件 id
     */
    @Schema(description = "文件 id")
    private String id;

    /**
     * 文件名
     */
    @Schema(description = "文件名")
    private String filename;

}

File: src/main/java/com/simonvonxcvii/turing/model/dto/UploadFileDTO.kt
package com.simonvonxcvii.turing.model.dto

/**
 * UploadFile DTO
 * 
 * @author Simon Von
 * @since 2023/4/3 10:52
 */
data class UploadFileDTO(
    /**
     * 文件 id
     */
    var id: Int? = null,

    /**
     * 文件名
     */
    var filename: String? = null
)
